README.md

```diff
@@ -102,6 +102,37 @@ Adding a new version means that you only need to:
 In both cases, when the version is finished, a release means adding the file to
 buildtest under `buildtest/tools/buildsystem/schemas`
 
+## Development Tips
+
+### Multiple Types
+
+See how [multiple types](https://cswr.github.io/JsonSchema/spec/multiple_types/) are handled.
+Basically, this snippet says that for shell the user can provide a string or an array (a list
+of strings or commands to execute). The variables `items`, and `minItems` apply only
+in the case that an array is used, and `minLength` applies to commands.
+
+```json
```

Comment:
I think this doc example is misleading. `shell` is of type string not array with the recent change.

Reply:
The README in the top level is only to provide examples to help developers, that's not the current implementation, but I undoubtedly will need to remember how to do that, which is why it's under a section called "Development Tips" and then "Multiple Types." The docs about the script command are in the README.md of the script folder, which is not this file.

Reply:
Make sense to me now.